client/core/trade.go

```diff
@@ -1191,7 +1191,7 @@ func (t *trackedTrade) shouldBeginFindRedemption(ctx context.Context, match *mat
 			coinIDString(t.wallets.fromAsset.ID, swapCoinID), t.wallets.fromAsset.Symbol, match, t.UID(), err)
 		return false
 	}
-	if spent {
+	if spent { // NOTE: spent may not be accurate for SPV wallet, so this should not be a requirement.
```

Comment:
You mean spent may be false for a spent swap, so we shouldn't rely on it to trigger find redemption? It seems looking below it doesn't really rely on spent being true, it attempts to find the redemption anyway if the contract has gotten enough confs.

Reply:
Exactly.  This is a note as to why it's an OR below, and never to change that.  The redemption search is the thing that'll make the spent determination.